### PR TITLE
Migrate to new maps and traffic controls

### PR DESCRIPTION
**Pull request by zhangk1551**
Changes corresponding to:
https://github.com/inverted-ai/torchdrivesim/pull/44

### CODE REVIEW

**Comment by zhangk1551** [comment]
test video: https://drive.google.com/file/d/14BoYys4fY-tXQ2Ah2fY6Pc0OdZ0rHZtL/view?usp=sharing

File: torchdriveenv/gym_env.py [hunk 1]
```diff
@@ -197,57 +196,19 @@ def close(self):
...
-                    yield_sign_poses.append(static_actors[id]['pos'])
-
-            if len(traffic_light_ids) > 0:
-                traffic_light_control = TrafficLightControl(location=location, pos=torch.stack(traffic_light_poses).unsqueeze(0), use_mock_lights=True, ids=traffic_light_ids)
-                traffic_light_states = dict(zip(static_actors.keys(), traffic_light_control.compute_state(0).squeeze()))
-                traffic_light_state_history = [{k:TrafficLightState(traffic_light_control.allowed_states[int(traffic_light_states[k])]) for k in traffic_light_states}]
-            else:
-                traffic_light_control = None
-                traffic_light_state_history = None
+        map_cfg = find_map_config(f"carla_{location}")
```
Comment (adscib) [inline torchdriveenv/gym_env.py]: Maybe include `carla_` in the location name? That will make it easier to incorporate non-carla maps in the future. Up to you.

File: torchdriveenv/gym_env.py [hunk 2]
```diff
@@ -343,7 +296,9 @@ def build_simulator(cfg: EnvConfig, location, ego_state, scenario=None, car_sequ
                 simulator=simulator, npc_mask=npc_mask, recurrent_states=[
                     recurrent_states],
                 rear_axis_offset=agent_attributes[..., 2:3], locations=[
-                    iai_location],
+                    f'carla:{":".join(location.split("_"))}'],
```
Comment (adscib) [inline torchdriveenv/gym_env.py]: Can you get it as `map_cfg.iai_location_name`?

**Review by adscib — COMMENTED** [review]
Looks good. I'll approve after the corresponding TDS PR is merged.

**Comment by zhangk1551** [comment]
Just found the video in this PR is also drive on the left. Let me check it.

**Review by adscib — APPROVED** [review]

**Pull request merged**